x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/components/hooks.tsx

```diff
@@ -111,6 +114,11 @@ export const getActionStatus = (status: ResponseActionStatus): string => {
   return '';
 };
 
+export const getCommand = (
+  command: ResponseActions
+): Exclude<ResponseActions, 'unisolate' | 'running-processes'> | 'release' | 'processes' =>
```

Comment:
I think you don't need to exclude `processes` here right? The type is `running-processes`

Reply:
Yeah I want to exclude it here cause I want to have a return type that looks like `'isolate' | 'kill-processes' | 'process' | 'release' | 'suspend-process'`

Reply:
Oh you are totally right, I totally missed the closing `>` next to `running-processes` and thought you were excluding all. 🙏